openedx/core/djangoapps/persistenttask/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FailedTask',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('task_name', models.CharField(max_length=255)),
+                ('task_id', models.CharField(max_length=255)),
+                ('argstring', models.CharField(max_length=4096, blank=True)),
+                ('kwargstring', models.CharField(max_length=4096, blank=True)),
+                ('exc', models.CharField(max_length=255)),
```

Comment:
Is `exc` intended to be just the name of the exception, or the detail message as well? If it's the latter, we may run into issues with a 255 character limit.